README.md

```diff
@@ -1,21 +1,25 @@
 # JQuery/Ajax Quiz
 
-Write your answer below each question. 
+Write your answer below each question.
 
 ### Question 1
-(a) What is a click handler? 
+(a) What is a click handler?
+A click handler triggers an event on a click to some target in the browser
 
-(b) Where in your JS file should you put it, and WHY? 
+(b) Where in your JS file should you put it, and WHY?
+should put it in the document.ready() part of your js so it only runs after everything on the page has loaded/is ready to be manipulated
 
 ### Question 2
-If you get the error `$ is undefined`, what does that mean and what could you do to fix it? 
+If you get the error `$ is undefined`, what does that mean and what could you do to fix it?
+this means that the variable/method has not been set to a value, which means you can fix it by giving it a value with the equals operator.
```

Comment:
In general, you're right. However, 99% of the time, `$ is undefined` specifically means that JQuery has not been loaded properly.